src/errorpage.cc

```diff
@@ -148,13 +148,12 @@ static void ValidateStaticError(const int page_id, const SBuf &inputLocation);
  \note  hard coded error messages are not appended with %S
  *      automagically to give you more control on the format
  */
-static const struct {
+struct DefaultMessages {
```

Comment:
```suggestion
struct HardCodedError {
```

or

```suggestion
struct HardCodedMessage {
```

The structure is about a _single_ message, and its purpose is to _hard-code_ the error body or its part. The "default" is how _some_ of these structures are used.

Reply:
Ok.